### PR TITLE
Finalizare

### PR DESCRIPTION
**Pull request by CLaurentiu55**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
         <div class="right-side fl" id="main">
-            <h2>Sidenav Push Example</h2>
-            <p>Click on the element below to open the side navigation menu, and push this content to the right.</p>
+            <div>
+                <h4>Recent repositories</h4>
+                <a href="#" class="fr">Show more . View all</a>
+            </div>
+            <div style="margin-top: 66px;  margin-bottom: 30px;">
+                <div class="card bkg-grey fl" style="width: 13rem; margin-right: 10px;">
+                    <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\php.png" class="card-img-top1" style="margin: 20px 20px -12px; z-index: 1;">
```
Comment (VKiril) [inline index.html:192]: please set proper path to image
it must be 'images/php.png'

File: index.html [hunk 2]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                    <div class="card-body" style="background-color: white;">
+                      <h5 class="card-text">GGGI Site</h5>
+                      <p class="grey">itworks-expert</p>
+                    </div>
+                </div>
+                <div class="card bkg-grey fl" style="width: 13rem; margin-right: 10px;">
+                    <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\c+.png" class="card-img-top1" style="margin: 24px 20px -12px; z-index: 1;">
```
Comment (VKiril) [inline index.html:199]: please set proper path to image

File: index.html [hunk 3]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                    <div class="card-body" style="background-color: white;">
+                      <h5 class="card-text">GGGI Amazon Bot</h5>
+                      <p class="grey">itworks-expert</p>
+                    </div>
+                </div>
+                <div class="card bkg-grey fl" style="width: 13rem; margin-right: 10px;">
+                    <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\code.png" class="card-img-top1" style="margin: 20px 20px -12px; z-index: 1;">
```
Comment (VKiril) [inline index.html:206]: please set proper path to image

File: index.html [hunk 4]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                    <div class="card-body" style="background-color: white;">
+                      <h5 class="card-text">TrickyBasketBall</h5>
+                      <p class="grey">Kiril Vasilita</p>
+                    </div>
+                </div>
+                <div class="card bkg-grey" style="width: 13rem; margin-right: 10px;">
+                    <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\code.png" class="card-img-top1" style="margin: 20px 20px -12px; z-index: 1;">
```
Comment (VKiril) [inline index.html:220]: please set proper path to image

File: index.html [hunk 5]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                            <th scope="col">Issue</th>
+                            <th scope="col">Status</th>
+                            <th scope="col">Assignee</th>
+                            <th scope="col">Actions</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        <tr>
+                            <td scope="row">
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\punct.png" class="fl" style="margin-top: 4px;">
```
Comment (VKiril) [inline index.html:247]: please set proper path to image

File: index.html [hunk 6]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                <p class="fl" style="padding: 0px 15px;">GGGI-122</p>
+                                <p class="fl">Am modificat datele din profilul personal si mi-a dat eroare</p>
+                            </td>
+                            <td style="padding: 0px 15px;">
+                                <span class="badge badge-primary">TESTING</span>
+                            </td>
+                            <td>
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\kv2.png" class="rounded-circle">
```
Comment (VKiril) [inline index.html:255]: please set proper path to image

File: index.html [hunk 7]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                    </button>
+                                    <div class="dropdown-menu dropdown-menu-right">
+                                        <button class="dropdown-item" type="button">Create branch</button>
+                                    </div>
+                                </div>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td scope="row">
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\punct.png" class="fl" style="margin-top: 4px;">
```
Comment (VKiril) [inline index.html:270]: please set proper path to image

File: index.html [hunk 8]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                <p class="fl" style="padding: 0px 15px;">GGGI-122</p>
+                                <p class="fl">Am modificat datele din profilul personal si mi-a dat eroare</p>
+                            </td>
+                            <td style="padding: 0px 15px;">
+                                <span class="badge badge-primary">TESTING</span>
+                            </td>
+                            <td>
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\kv2.png" class="rounded-circle">
```
Comment (VKiril) [inline index.html:278]: please set proper path to image

File: index.html [hunk 9]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                    </button>
+                                    <div class="dropdown-menu dropdown-menu-right">
+                                        <button class="dropdown-item" type="button">Create branch</button>
+                                    </div>
+                                </div>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td scope="row">
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\punct.png" class="fl" style="margin-top: 4px;">
```
Comment (VKiril) [inline index.html:293]: please set proper path to image

File: index.html [hunk 10]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                <p class="fl" style="padding: 0px 15px;">GGGI-122</p>
+                                <p class="fl">Am modificat datele din profilul personal si mi-a dat eroare</p>
+                            </td>
+                            <td style="padding: 0px 15px;">
+                                <span class="badge badge-secondary">Done</span>
+                            </td>
+                            <td>
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\kv2.png" class="rounded-circle">
```
Comment (VKiril) [inline index.html:301]: please set proper path to image

File: index.html [hunk 11]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                    </button>
+                                    <div class="dropdown-menu dropdown-menu-right">
+                                        <button class="dropdown-item" type="button">Create branch</button>
+                                    </div>
+                                </div>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td scope="row">
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\punct.png" class="fl" style="margin-top: 4px;">
```
Comment (VKiril) [inline index.html:316]: please set proper path to image

File: index.html [hunk 12]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                <p class="fl" style="padding: 0px 15px;">GGGI-122</p>
+                                <p class="fl">Am modificat datele din profilul personal si mi-a dat eroare</p>
+                            </td>
+                            <td style="padding: 0px 15px;">
+                                <span class="badge badge-primary">TESTING</span>
+                            </td>
+                            <td>
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\kv2.png" class="rounded-circle">
```
Comment (VKiril) [inline index.html:324]: please set proper path to image

File: index.html [hunk 13]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                    </button>
+                                    <div class="dropdown-menu dropdown-menu-right">
+                                        <button class="dropdown-item" type="button">Create branch</button>
+                                    </div>
+                                </div>
+                            </td>
+                        </tr>
+                        <tr>
+                            <td scope="row">
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\punct.png" class="fl" style="margin-top: 4px;">
```
Comment (VKiril) [inline index.html:339]: please set proper path to image

File: index.html [hunk 14]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
+                                <p class="fl" style="padding: 0px 15px;">GGGI-122</p>
+                                <p class="fl">Am modificat datele din profilul personal si mi-a dat eroare</p>
+                            </td>
+                            <td style="padding: 0px 15px;">
+                                <span class="badge badge-primary">TESTING</span>
+                            </td>
+                            <td>
+                                <img src="C:\Users\GiGaByTe\Pictures\Camera Roll\kv2.png" class="rounded-circle">
```
Comment (VKiril) [inline index.html:347]: please set proper path to image

File: index.html [hunk 15]
```diff
@@ -105,17 +105,16 @@
                             </span>
                         </div>
                     </div>
-                    <div style="margin-left: 5px;
-                    margin-top: -30px;">
+                    <div style="margin-left: 5px; margin-top: -30px;">
```
Comment (VKiril) [inline index.html:108]: remove all inline styles

File: index.html [hunk 16]
```diff
@@ -184,8 +183,183 @@ <h6 class="pl25">
...
         </div>
         <div class="right-side fl" id="main">
-            <h2>Sidenav Push Example</h2>
-            <p>Click on the element below to open the side navigation menu, and push this content to the right.</p>
+            <div>
+                <h4>Recent repositories</h4>
+                <a href="#" class="fr">Show more . View all</a>
+            </div>
+            <div style="margin-top: 66px;  margin-bottom: 30px;">
+                <div class="card bkg-grey fl" style="width: 13rem; margin-right: 10px;">
```
Comment (VKiril) [inline index.html:191]: test

Comment (CLaurentiu55) [reply index.html:191]: what do you mean

File: index.html (hunk 1 above)
Comment (CLaurentiu55) [reply index.html:192]: Done

**Pull request merged**